test/pageobjects/Login.page.js

```diff
@@ -0,0 +1,50 @@
+
+import BasePage from '../pageobjects/base.page'
+
+class Login extends BasePage{
+
+    //WebElements
+    get principal(){return $('.align-content')}
+    get tittle(){return $('.login-logo')}
+    get usernamein(){ return $('#username') }
+    get passwordin(){return $('#password')}
+    get rememberme(){return $('#remember-me')}
+    get signin(){return $('#submit')}
+    get signuphere(){return $('=Sign Up Here')}
+    get flash() {return $('.badge')}
+
+    /**Metodos para iniciar sesion en Digital bank
+     * @param {String} user
+     * @param {String} password
+     * 
+     */
+    async loguearclick(username,password){
```

Comment:
Este nombre, en mi opinion, tiene algo de sobre-abreviacion. Podrias mejorarlo con un: `loguearseConClickEnBotonSubmit()` (puede parecer largo, pero los beneficios de la claridad que aporta, en mi opinion, son mucho mayores a los posibles "costos" de la longitud). Hay un principio en desarrollo de tests llamado DAMP (descriptive and meaningful phrases) que se suele usar al implementarlos, que es basicamente preferir implementaciones faciles de leer e interpretar, aunque a veces implique mas duplicacion (violar otro principio muy fuerte en software que es el DRY) (que no es el caso aca)

Ref: https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests